tensorflow/BUILD

```diff
@@ -358,6 +358,14 @@ config_setting(
     },
 )
 
+config_setting(
```

Comment:
It should be used from the .tf_configure.bazelrc file. Config is enabling this. It was first introduced to conditionally construct the local_tensorrt_repository and add tensorrt to the pywrap_tensorflow. But since then we found a workaround the issue and probably can remove this.